test/models/viewmodels/decoder/RoeCodeSpec.scala

```diff
@@ -41,7 +40,7 @@ class RoeCodeSpec extends UnitSpec {
         PrelodgePrefix
       )
 
-      RoeCode.codes mustBe expectedCodes
+      RoeCode.codes mustEqual expectedCodes
```

Comment:
Why `mustEqual`?

Reply:
Changed back to `mustBe` in a separate commit. In this case there is no difference in functionality, but should compile slightly faster.